source/D2Game/include/GAME/Clients.h

```diff
@@ -183,7 +183,7 @@ struct D2ClientStrc
 	int32_t nSaveHeaderSize;					//0x180
 	uint32_t unk0x184[2];						//0x184
 	DWORD nSaveChecksum;						//0x18C
-	DWORD unk0x190;								//0x190
+	DWORD nPlayerMark;							//0x190
```

Comment:
this is 2 DWORD's in length

Reply:
Do you know what the second word is? I can only see it as "reserved"

Reply:
Off the top of my head its a 64bit value used as a GUID within ladder ranking (hence it being 2 DWORD's long).

Reply:
Looking at GAME_ReceiveDatabaseCharacter, it seems to indeed be of a length of 2 DWORDS